test/web-platform-tests/index.js

```diff
@@ -145,6 +146,14 @@ describe("Web Platform Tests", () => {
     "html/dom/dynamic-markup-insertion/document-writeln/document.writeln-02.html",
     "html/dom/dynamic-markup-insertion/document-writeln/document.writeln-03.html",
     "html/dom/elements/global-attributes/classlist-nonstring.html",
+    "html/dom/elements/global-attributes/custom-attrs.html",
+    "html/dom/elements/global-attributes/data_unicode_attr.html",
+    // "html/dom/elements/global-attributes/dataset.html", // we only support HTML elements
+    "html/dom/elements/global-attributes/dataset-delete.html",
+    "html/dom/elements/global-attributes/dataset-enumeration.html",
+    "html/dom/elements/global-attributes/dataset-get.html",
+    // "html/dom/elements/global-attributes/dataset-prototype.html", // tests Object.prototype
```

Comment:
Yeah, the dataset object's prototype is the Object.prototype from another <del>universe</del> context, not the window one.